explorer/ast/member.h

```diff
@@ -25,6 +26,16 @@ struct NamedValue {
   Nonnull<const Value*> value;
 };
 
+// A IndexedValue represents a value identified by an index, such as a tuple
+// field
+struct IndexedValue {
+  // The field index.
+  size_t index;
```

Comment:
```suggestion
  int index;
```

I believe `int` is more consistent with style and surrounding code: https://google.github.io/styleguide/cppguide.html#Integer_Types

Reply:
That's going to (somewhat) limit our capacity to address content beyond 2^31 for tuples, and needs `size_t` casts here and there. But consistency might be preferred. Change done, can always be reverted if needed.